tasks/unit_tests/omnibus_tests.py

```diff
@@ -89,9 +89,9 @@ def test_successful_cache_hit(self):
         self.assertRunLines(
             [
                 # We copied the cache from remote cache
-                r'aws s3 cp (\S* )?s3://omnibus-cache/builds/\w+/slug /tmp/omnibus-git-cache-bundle',
+                r'aws s3 cp (\S* )?s3://omnibus-cache/builds/\w+/slug /tmp/\S+/omnibus-git-cache-bundle',
```

Comment:
Given that technically there's no guarantee that `TemporaryDirectory()` will give us a folder under `/tmp` specifically, it might be worth relaxing these assertions to something like:

```suggestion
                r'aws s3 cp (\S* )?s3://omnibus-cache/builds/\w+/slug \S+/omnibus-git-cache-bundle',
```